bin/s1_read.py

```diff
@@ -27,3 +27,6 @@
 
     for i, burst in enumerate(bursts):
         print(burst.burst_id, burst.center)
+    burst.slc_to_file('burst.slc')
+    burst.slc_to_file('burst.tif', 'GTiff')
```

Comment:
GTiff may not be our favorite format here. First the bursts are not geocoded so using GeoTiff may not be needed. Second we commonly use mdx to display the SLC data and mdx is happy with flat binary files. Using ENVI will make a simple flat binary file and that's why I prefer until we have a more advanced viewer working as fast as mdx.

Reply:
I forgot to write comments. Fixed in 06b636e.

I included the 3 calls to `slc_to_file` at the bottom on `s1_read.py` to test if the 3 write modes worked. https://github.com/opera-adt/sentinel1-reader/blob/06b636e50e21170310f52444f4264bc697c108fd/bin/s1_read.py#L33-L38.